data/importers/page.md

```diff
@@ -212,162 +212,162 @@ For each importer, we will cover only columns of interest.
 * Comma separated list of Fleet IDs - 5, 15, 25, 30. IDs can be retrieved from Export or VDS.
 ---
 * Altitude
-* Cruise altitude in the feet format (e.g. 35000).
-*Note: Can be Zero, in case you want the Simbrief to decide the cruise altitude*
+* Cruise altitude in following formats: 35000, FL350, /350 or 350
+  *Note: Can be blank, in case you want the Simbrief to decide the cruise altitude, but the column is required*
```

Comment:
We have a misunderstanding. The spaces need to be after `or 350` so it's `or 350  ` and the `*Note[...]` does not need any spacing at all. The entire thing would look like this:
```
* Cruise altitude in following formats: 35000, FL350, /350 or 350  
*Note: Can be blank, in case you want the Simbrief to decide the cruise altitude, but the column is required*
```